### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bingo4508-test**

### CODE REVIEW

**Review by chialunwu — APPROVED** [review]
ok

**Review by chialunwu — COMMENTED** [review]
1

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Test1
+# Test2
```
Comment (chialunwu) [inline README.md:1]: test (edited)

File: package.json [hunk 2]
```diff
@@ -1 +1,2 @@
 1
+2
```
Comment (chialunwu) [inline package.json:2]: test2

**Review by chialunwu — CHANGES_REQUESTED** [review]

**Comment by chialunwu** [comment]
yoyo

File: README.md (hunk 1 above)
Comment (chialunwu) [reply README.md:1]: yo

File: package.json (hunk 2 above)
Comment (chialunwu) [reply package.json:2]: no

File: README.md (hunk 1 above)
Comment (chialunwu) [reply README.md:1]: test

File: package.json (hunk 2 above)
Comment (chialunwu) [reply package.json:2]: test2

File: README.md (hunk 1 above)
Comment (chialunwu) [reply README.md:1]: test

Comment (chialunwu) [reply README.md:1]: yoyo

Comment (chialunwu) [reply README.md:1]: test111!!

Comment (chialunwu) [reply README.md:1]: test

File: package.json (hunk 2 above)
Comment (chialunwu) [reply package.json:2]: test

**Review by chialunwu — CHANGES_REQUESTED** [review]
YO

**Review by chialunwu — APPROVED** [review]
test

File: README.md (hunk 1 above)
Comment (bingo4508-test) [reply README.md:1]: YO

File: package.json [hunk 3]
```diff
@@ -1 +1,2 @@
 1
```
Comment (bingo4508-test) [inline package.json:1]: YO

Comment (bingo4508-test) [inline package.json:1]: hello

**Pull request merged**